lib/internal/bootstrap_node.js

```diff
@@ -474,6 +478,7 @@
   }
 
   NativeModule._source = process.binding('natives');
+  NativeModule.preloadModuleWrap = null;
```

Comment:
I just went with the `null` so there's no extra call in the vast majority of cases when it's not in-use. Probably not a big deal either way though.